translate/translate.go

```diff
@@ -18,28 +18,49 @@ type Content struct {
 	Slug        string
 }
 
-func (tc *TranslationContext) Translate(item mapper.Item, itemType mapper.Type) (fileName string, content string) {
-
-	rawContent := tc.convertContent(item.Fields, itemType.Fields)
-	if tc.TransConfig.Encoding == "yaml" {
-		content = rawContent.ToYaml()
-	} else {
-		content = rawContent.ToToml()
+// UnionValuesAndOverride takes a defaults and an overrides map and assigns any missing
+// values from the defaults to the overrides map.
+func (tc *TranslationContext) UnionValuesAndOverride(itemDefault map[string]interface{}, itemOverride map[string]interface{}) (combinedItem map[string]interface{}) {
```

Comment:
Style stuff: I think `Override` is intuitive and not needed in the function name. `MergeMaps` might be a little more succinct